pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -348,7 +348,18 @@ func (r *Replica) RangeFeed(
 func (r *Replica) getRangefeedProcessorAndFilter() (rangefeed.Processor, *rangefeed.Filter) {
 	r.rangefeedMu.RLock()
 	defer r.rangefeedMu.RUnlock()
-	return r.rangefeedMu.proc, r.rangefeedMu.opFilter
+	p := r.rangefeedMu.proc
```

Comment:
Add a comment on `proc` that this should always be obtained via `getRangefeedProcessorAndFilter`. Looking at the accesses, it seems true enough today - the remaining reads are "very internal":

<img width="469" alt="image" src="https://github.com/user-attachments/assets/7e744fe1-83e8-4281-a41a-288e4c796ebd" />